src/bezier.typ

```diff
@@ -318,20 +315,15 @@
   }
 }
 
-/// Shorten curve by distance d. This keeps the curvature of the
-/// curve by finding new values along the original curve.
-/// If d is positive the curve gets shortened by moving s
-/// closer to e, if d is negative, e is moved closer to s.
-/// The points s and e are moved along the curve, keeping the
-/// kurves curvature the same (the control points get recalculated).
+/// Shorten curve by distance `d`. This keeps the curvature of the curve by finding new values along the original curve. If `d` is positive the curve gets shortened by moving `s` closer to `e`, if `d` is negative, `e` is moved closer to `s`. The points `s` and `e` are moved along the curve, keeping the curve's curvature the same (the control points get recalculated).
```

Comment:
Why such super long lines? I prefer a max line length of 110 chars.

Reply:
Oh I have word wrap turned on so I don't notice how long they get ':) I can change that if you really want.

Reply:
Not fixing as a change in the regex is required for parameter descriptions which makes this difficult...